modules/developers-guide/pages/default-wallet.adoc

```diff
@@ -144,42 +352,105 @@ The command returns a list of principals similar to the following:
   },
 )
 ....
+////
 
 [[wallet-deauthorize]]
 == Remove authorization for a custodian
 
-Use the `deauthorize` method to remove a principal as a custodian for a cycles wallet.
+You can use the `+dfx wallet deauthorize+` command or the `+deauthorize+` method to remove a principal as a custodian for a cycles wallet.
```

Comment:
this will also remove them from the controller role if they have that designation